substrafl/strategies/strategy.py

```diff
@@ -14,7 +14,7 @@
 
 
 class Strategy(ABC):
-    """Base strategy to be inherited from substrafl strategies."""
+    """Base strategy to be inherited from SubstraFL strategies."""
 
     def __init__(self, *args, **kwargs):
```

Comment:
We probably want to enforce at that level that `Strategy` always have an `Algo` parameter

Reply:
I did that in the begining, but as we save args and kwargs, I went backward and removed it. But your comment make me realize that I could just add the algo to "self.kwargs" in the __init__, and enforce the algo. I'll try this way !